lib/ability.rb

```diff
@@ -28,7 +28,7 @@ def initialize(user, resource, controller)
         can :manage, [AccountPriceGroupMember, UserPriceGroupMember]
       end
 
-      if resource.global? && (user.user_roles.map(&:role) & UserRole.facility_roles).any?
+      if resource.global? && (user.user_roles.map(&:role) & UserRole.facility_roles).any? && resource.admin_editable?
```

Comment:
what about `editable_global_group?` or something like that? I don't like using cancer center as part of the name since it's specific to northwestern (although, we do end up using that string elsewhere, but I'd like to move away from it).